### DISCUSSION TITLE
Photoprism → Immich: Migrating favorites and albums

### DISCUSSION THREAD

**Discussion by v411e** [immich-app/immich#4701] [Community Guides]
I recently switched from Photoprism to Immich. To make the migration easier, I wrote a small CLI tool that migrates the albums and favorites from Photoprism to Immich.

Feel free to use it or create a PR if something is missing.

→ https://github.com/v411e/ppim-migrator

**Comment by alextran1502** [comment]
Thank you, I will pin this to the discussion for people to find the information easier

**Comment by mirisbowring** [comment]
Oh man! Would have needed this 2 months ago where i did this fully manual 😆

**Reply by ipnetxyz** [reply]
Same here, but I'm happy the ones which will switch next have this tool to help 👍
